### PR TITLE
✨Add LabeledInput

### PR DESCRIPTION
**Pull request by jujuzitroni**
in preperation for #1

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -22435,7 +22435,6 @@
       "integrity": "sha512-qOtwgiqI3LMqT0eXYNV6ykp7qSu0LQGeXxy3wOBGuDDqAizfgnAjomYEWGFcyKp5ahV7HCRCjxbixAklFPUmyw==",
       "dev": true,
       "requires": {
-        "@babel/core": "^7.12.10",
```
Comment (lmachens) [inline package-lock.json]: This change is not related. Can you remove it from changes?

File: src/app/components/LabeledInput/LabeledInput.module.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600&display=swap');
```
Comment (lmachens) [inline src/app/components/LabeledInput/LabeledInput.module.css]: This font (in different font-weights) is loaded in `globals.css`. If it is required to have additional font-weights, please update the existing import.

File: src/app/components/LabeledInput/LabeledInput.module.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+.LabeledInput_label {
+  color: var(--text-color);
+  display: grid;
+  gap: 0.7em;
+  text-transform: uppercase;
+  font-family: 'Montserrat', sans-serif;
```
Comment (lmachens) [inline src/app/components/LabeledInput/LabeledInput.module.css]: Is it required to change the font-family? This is already the default font.
If it is required, please use `--font-family`.

File: src/app/components/LabeledInput/LabeledInput.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+export default LabeledInput;
+
+// Beispiel react input element:
+// {
+//   /* <label>
+//     Name:
+//     <input type="text" name="name" />
+//   </label> */
+// }
```
Comment (lmachens) [inline src/app/components/LabeledInput/LabeledInput.tsx]:
```suggestion
```

File: src/app/components/LabeledInput/LabeledInput.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  return (
+    <label className={styles.LabeledInput_label}>
+      {children}
+      <input
+        className={styles.LabeledInput_input}
+        type="text"
+        placeholder={`${children}` + ' here'}
+      >
+        {}
+      </input>
```
Comment (lmachens) [inline src/app/components/LabeledInput/LabeledInput.tsx]: An `input` doesn't have children. Can you write it as a self-closing void element?

File: src/app/components/LabeledInput/LabeledInput.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode } from 'react';
+import styles from './LabeledInput.module.css';
+
+type LabeledInputProps = {
+  children: ReactNode;
```
Comment (lmachens) [inline src/app/components/LabeledInput/LabeledInput.tsx]: You are using `children` as part of the placeholder. Thus, `children` is only allowed to be a string.
But it's possible to add other HTML elements as children because of the type `ReactNode`.

Anyway, I would rework the properties and split it in two:

```suggestion
  label: string;
  placeholder: string;
```

If you like to allow more complex labels, use `label: ReactNode`.

File: src/app/components/LabeledInput/LabeledInput.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import LabeledInput from './LabeledInput';
+
+export default {
+  title: 'Component/LabeledInput',
+  component: LabeledInput,
+};
+
+export const email = (): JSX.Element => <LabeledInput>email</LabeledInput>;
```
Comment (lmachens) [inline src/app/components/LabeledInput/LabeledInput.stories.tsx]:
```suggestion
export const email = (): JSX.Element => <LabeledInput>Email</LabeledInput>;
```

File: src/app/components/LabeledInput/LabeledInput.stories.tsx [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+export const password = (): JSX.Element => (
+  <LabeledInput>password</LabeledInput>
```
Comment (lmachens) [inline src/app/components/LabeledInput/LabeledInput.stories.tsx]:
```suggestion
  <LabeledInput>Password</LabeledInput>
```

File: src/app/components/LabeledInput/LabeledInput.module.css [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+  font-weight: 600;
+  font-size: 0.8em;
+}
+
+.LabeledInput_input {
```
Comment (lmachens) [inline src/app/components/LabeledInput/LabeledInput.module.css]: I like to use the default BEM convention in this project. Can you rename it?
https://en.bem.info/methodology/quick-start/

**Review by lmachens — COMMENTED** [review]
Good job! I like it :-). 

Could you rename the title of this pull request?

**Review by Dev-dfm — COMMENTED** [review]
lgtm 👍

**Review by tmh9x — APPROVED** [review]

File: src/app/components/LabeledInput/LabeledInput.module.css (hunk 9 above)
Comment (lmachens) [reply src/app/components/LabeledInput/LabeledInput.module.css]: There is an interesting article:
https://github.com/lmachens/photoplay/pull/26#discussion_r639660707

**Review by lmachens — APPROVED** [review]
\^o^/
